lib/pharos/config_schema.rb

```diff
@@ -5,6 +5,14 @@
 
 module Pharos
   class ConfigSchema
+    DEFAULT_DATA = {
+      'hosts' => [],
+      'api' => {},
+      'network' => {},
+      'authentication' => {},
+      'kube_proxy' => {}
+    }.freeze
```

Comment:
We need to find better way in future for this.. without placeholder values dry-struct does not set defaults to embedded structs.